### PR TITLE
[Android] Added Edit profile screen

### PR DESCRIPTION
**Pull request by AndriiLatash**
https://trello.com/c/rdhzX7cM/6-create-edit-profile-screen

### CODE REVIEW

**Review by kostasante — APPROVED** [review]

File: androidApp/src/main/java/com/gorosoft/bookme/now/android/ui/main_screen/profile/edit_profile/EditProfileScreen.kt [hunk 1]
```diff
@@ -0,0 +1,592 @@
...
+                    DateOfBirthInput(
+                        dateOfBirth = profileState.dateOfBirthText,
+                        onDateClick = { showDateDialog = true },
+                        focusManager = focusManager,
+                    )
+
+                    var email = profileState.email
+                    var isEmailValid by remember { mutableStateOf(true) }
+                    EmailInput(
+                        email = profileState.email,
```
Comment (AHuminskyi) [inline androidApp/src/main/java/com/gorosoft/bookme/now/android/ui/main_screen/profile/edit_profile/EditProfileScreen.kt:166]: This is not working. You can delete this line. The real work is in `onEmailInputted`

File: androidApp/src/main/java/com/gorosoft/bookme/now/android/ui/main_screen/profile/edit_profile/EditProfileScreen.kt [hunk 2]
```diff
@@ -0,0 +1,592 @@
...
+                            email = input
+                            isEmailValid =
+                                android.util.Patterns.EMAIL_ADDRESS.matcher(input).matches()
+                            onEmailInputted(input)
+                        },
+                        isEmailValid = isEmailValid,
+                        focusManager = focusManager,
+                    )
+
+                    CountryInput(
```
Comment (AHuminskyi) [inline androidApp/src/main/java/com/gorosoft/bookme/now/android/ui/main_screen/profile/edit_profile/EditProfileScreen.kt:177]: Let's comment this CountryInput sience we don't have countries list.

File: androidApp/src/main/java/com/gorosoft/bookme/now/android/ui/main_screen/profile/edit_profile/EditProfileScreen.kt [hunk 3]
```diff
@@ -0,0 +1,592 @@
...
+                        country = profileState.country,
+                        onCountryFieldClick = {
+                            keyboardController?.hide()
+                            coroutineScope.launch { genderBottomSheetState.show() }
+                        },
+                        focusManager = focusManager,
+                    )
+
+                    PhoneInput(
```
Comment (AHuminskyi) [inline androidApp/src/main/java/com/gorosoft/bookme/now/android/ui/main_screen/profile/edit_profile/EditProfileScreen.kt:186]: Let's comments this `PhoneInput` while we don't have phones and countires list.

File: androidApp/src/main/java/com/gorosoft/bookme/now/android/ui_models/EditProfileUiModel.kt [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import com.gorosoft.bookme.now.domain.models.UserGenderType
+import kotlinx.parcelize.Parcelize
+import java.time.LocalDate
+
+@Parcelize
+data class EditProfileUiModel(
+    val firstName: String = "",
+    val secondName: String = "",
+    val email: String = "",
+    val country: UserGenderType? = null,
```
Comment (AHuminskyi) [inline androidApp/src/main/java/com/gorosoft/bookme/now/android/ui_models/EditProfileUiModel.kt:14]: let's comment this and phone too.

File: androidApp/src/main/java/com/gorosoft/bookme/now/android/ui/main_screen/profile/edit_profile/EditProfileViewModel.kt [hunk 5]
```diff
@@ -0,0 +1,50 @@
+package com.gorosoft.bookme.now.android.ui.main_screen.profile.edit_profile
+
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import com.gorosoft.bookme.now.android.ui_models.EditProfileUiModel
+import com.gorosoft.bookme.now.domain.models.UserGenderType
+import java.time.LocalDate
+
+private const val ProfileKey = "EditYourProfile"
```
Comment (AHuminskyi) [inline androidApp/src/main/java/com/gorosoft/bookme/now/android/ui/main_screen/profile/edit_profile/EditProfileViewModel.kt:9]: It's not a mistake, but you forgot to rename `ProfileKey` to `EditProfileKey` to follow the general class idea :)

File: androidApp/src/main/java/com/gorosoft/bookme/now/android/ui/main_screen/profile/edit_profile/EditProfileScreen.kt [hunk 6]
```diff
@@ -0,0 +1,592 @@
...
+                },
+            )
+        },
+        content = {
+            val scrollState = rememberScrollState()
+
+            Box(modifier = modifier.fillMaxSize()) {
+                Column(
+                    modifier = modifier
+                        .fillMaxWidth()
```
Comment (AHuminskyi) [inline androidApp/src/main/java/com/gorosoft/bookme/now/android/ui/main_screen/profile/edit_profile/EditProfileScreen.kt:128]: It's not a mistake just an option. Instade of using two functions `fillMaxWidth` and `fillMaxHeight` you can use function `fillMaxSize` that will do the same but in one line.

**Review by AHuminskyi — APPROVED** [review]

**Pull request merged**